sky/catalog/data_fetchers/fetch_runpod.py

```diff
@@ -0,0 +1,466 @@
+import os
+import sys
+import traceback
+from typing import Any, Dict, List, Optional, Union
+
+import pandas as pd
+import runpod
+from runpod.api import graphql
+
+# The API currently returns a dynamic number of vCPUs per pod that
+# changes frequently (less than 30 mins)
+# Therefore we hard code a default number of vCPUs from:
+# 1. The previous catalog, if the GPU exists there
+# 2. Or if not, the pricing page https://www.runpod.io/pricing
+# 3. Otherwise, the minimum of the returned# vCPU count from the API
+# The max count of GPUs per pod is set to 8 apart from A40 at 10
+DEFAULT_MAX_GPUS = 8
+DEFAULT_GPU_INFO: Dict[str, Dict[str, Union[int, float]]] = {
+    'A100-80GB': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'A100-80GB-SXM': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'A30': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'A40': {
+        'vcpus': 9.0,
+        'max_count': 10
+    },
+    'B200': {
+        'vcpus': 36.0,
+        'max_count': 8
+    },
+    'H100': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'H100-NVL': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'H100-SXM': {
+        'vcpus': 20.0,
+        'max_count': 8
+    },
+    'H200-SXM': {
+        'vcpus': 20.0,
+        'max_count': 8
+    },
+    'L4': {
+        'vcpus': 4.0,
+        'max_count': 8
+    },
+    'L40': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'L40S': {
+        'vcpus': 16,
+        'max_count': 8
+    },
+    'MI300X': {
+        'vcpus': 24.0,
+        'max_count': 8
+    },
+    'RTX2000-Ada': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTX3070': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'RTX3080': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'RTX3080-Ti': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'RTX3090': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'RTX3090-Ti': {
+        'vcpus': 12.0,
+        'max_count': 8
+    },
+    'RTX4000-Ada': {
+        'vcpus': 9.0,
+        'max_count': 8
+    },
+    'RTX4090': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'RTX5000-Ada': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTX5080': {
+        'vcpus': 5.0,
+        'max_count': 8
+    },
+    'RTX5090': {
+        'vcpus': 5.0,
+        'max_count': 8
+    },
+    'RTX6000-Ada': {
+        'vcpus': 14.0,
+        'max_count': 8
+    },
+    'RTXA4000': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTXA4500': {
+        'vcpus': 12.0,
+        'max_count': 8
+    },
+    'RTXA5000': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTXA6000': {
+        'vcpus': 4.0,
+        'max_count': 8
+    },
+    'RTXPRO6000': {
+        'vcpus': 14.0,
+        'max_count': 8
+    },
+    'RTXPRO6000-MaxQ': {
+        'vcpus': 18.0,
+        'max_count': 8
+    },
+    'RTXPRO6000-WK': {
```

Comment:
why do you do RTXPRO6000 instead of RTX-PRO-6000, the later name updates would suggest that spaces are replaced with - but in this it appears to be using a different heuristic. Perhaps it would be a good idea to centralize and standardize how the names of GPUs are defined. Then allow for each fetcher script to define its own mapping from the centralized GPU name list and the gpu name list for that particular vendor.